### PR TITLE
chore: add dependabot config file

### PR DESCRIPTION
**Pull request by AbdelrahmanHafez**
This will make dependabot check daily for new versions of packages that we use and create PRs for each dependency.

### CODE REVIEW

**Comment by vkarpov15** [comment]
I'd strongly prefer not to do this. My inbox just can't handle that much noise. And most of the time it isn't worth it for us to upgrade packages.

**Comment by Uzlopak** [comment]
Maybe a compromise:

Fastify Framework changed recently all their dependabot configurations to report only monthly about new packages.
So how about also set it to monthly. Should be a good balance between having an eye on new packages and not detonating @vkarpov15 Inbox :D.

**Comment by AbdelrahmanHafez** [comment]
@Uzlopak 's suggestion works well for me.
What do you think? @vkarpov15

**Comment by vkarpov15** [comment]
Is there a way to mute updates for certain packages? For example, I don't want to go through the effort of updating `dox` yet. There's just too much risk for too much of the docs, without any way of testing the upgrade automatically. I also don't want a bot sending me reminders every month to upgrade `dox`.

**Comment by Uzlopak** [comment]
Should be possible

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#ignore

**Comment by Uzlopak** [comment]
But maybe we should first update non critical packages before we activate dependabot and get annoyed from the beginning.

**Comment by vkarpov15** [comment]
Works for me. Can we also change it to only run quarterly, or at least monthly? Every day is a bit aggressive.

File: .github/.dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: "daily"
```
Comment (Uzlopak) [inline .github/.dependabot.yml:6]:
```suggestion
    interval: "monthly"
```

**Comment by Uzlopak** [comment]
According to the dependabot docs only monthy is possible.

**Review by vkarpov15 — APPROVED** [review]
Thanks :+1:

**Comment by AbdelrahmanHafez** [comment]
Happy to help 👍

**Pull request merged**